compiler/luci/import/include/luci/Import/CircleReader.h

```diff
@@ -48,57 +48,80 @@ MirrorPadMode luci_mirrorpad_mode(const circle::MirrorPadMode mode);
 luci::CircleFullyConnected::WeightsFormat
 luci_weights_format(const circle::FullyConnectedOptionsWeightsFormat weights_format);
 std::unique_ptr<CircleQuantParam>
-luci_quantparam(const circle::QuantizationParametersT *quantization);
+luci_quantparam(const circle::QuantizationParameters *quantization);
 
 /// @brief Copy common tensor attributes such as name, type, etc. to node.
-void copy_tensor_attributes(const circle::TensorT &tensor, CircleNode *node);
+void copy_tensor_attributes(const circle::Tensor *tensor, CircleNode *node);
+
+/**
+ * @brief Wrapper to use flatbuffers::Vector pointer as std::vector entity
+ */
+template <typename T> class VectorWrapper
```

Comment:
this too about `VectorWrapper`

Reply:
I made this struct to minimize code changes. Let me explain the reason with some simple example! 
Currently, we use some methods of reader (e.g. `inputs()`), which returns references to `std::vector`, which created by model's `UnPack`.

https://github.com/Samsung/ONE/blob/393385e4a29627b7143d509866b4452bbba0de9f/compiler/luci/import/src/Importer.cpp#L76

But usage direct access API return **pointer** to vector, which could be equal to `nullptr`. So, before using of range based for loop we need make each time following actions:
1. Check result is not `nullptr`
2. Deref pointer.  
3. replace all `at(n)` usages to `Get()` usages.

In order to minimize changes, I made safe `std::vector`-like wrapper, which helps to avoid many `nullptr` checks and code replacements.